docs/topics/api/index.rst

```diff
@@ -2,8 +2,26 @@
 External API
 ============
 
-This is documentation for the API available at `/api/v3/`.
+This shows you how to use the `addons.mozilla.org <https://addons.mozilla.org/en-US/firefox/>`_
```

Comment:
Yes because otherwise Sphinx's `linkcheck` will complain that the documented URL is redirected.